utopian-io/v2.utopian.io
# Fix the issue with project editing rights

### Verify that
Git checklist
- [X] the branch name follows the git flow naming convention + label + UT reference if applicable 

### My PR doesn't concern a Jira task
Fix the issue with project editing rights when a user is a collaborator

### **Test plan**

_Edit as an owner_

* Edit a field from one of your projects

**Expected result:** You should be able to update

_Edit as a collaborator with project role_

* Edit a field from one of your projects

**Expected result:** You should be able to update
